lib/cmsis_rtos_v1/cmsis_thread.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __CMSISTHREAD_H__
+#define __CMSISTHREAD_H__
+
+#include <kernel.h>
+#include <cmsis_os.h>
+
+struct cmsis_thread {
+	struct k_mem_block block;
+	struct k_thread thread;
+};
+
+#endif
```

Comment:
A whole header with a copyright declaration, license ID, include guards and two include files (one of which is completely superfluous, btw: there are no symbols from cmsis_os referenced here) just to declare a two-field struct.

Is this header an official CMSIS thing that external apps will expect to find?  If so, then I guess that's unavoidable.  But would be nice to fold this together with other files.

Reply:
Thanks for pointing this out. It was originally created to hold a little more but then got reduced to only 1 struct. Will move this to the corresponding .c file.